gateway/test/src/test/resources/vertx3/conf-es.json

```diff
@@ -181,5 +181,3 @@
   // When reporting a API's endpoint, prefer secure (i.e. https)
   "preferSecure": false
 }
-
-}
```

Comment:
Is this okay?

Reply:
There was a syntax error in the file before with an unmatched end brace; it was evidently being ignored in previous versions for whatever reason. 

If you check the file, you will see the JSON syntax is now valid.

I have no idea why it worked previously.